### PR TITLE
[ORPH-7] Improved handling of missing server credentials for login.

### LINKED ISSUE #7: Improve handling of missing login credentials
Currently, if a `repoId` is provided that does not exist within the `settings.xml` servers collection, or if the provided `repoId` is missing either the `username` or `password`, orphedomos will fail in an unexpected, unclear manner.  Login will continue to be attempted with empty strings in place of the missing credentials.  We would prefer to handle this case more clearly, and provide appropriate validation of the retrieved credentials before attempting login.

### PR DESCRIPTION
**Pull request by peter-mcclonski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,18 +143,6 @@ where the password is stored in an encrypted way, but simply provided as plain-t
 
 *Default:* `""`
 
-#### Snyk Token
-
-*.m2/settings:*
```
Comment (peter-mcclonski) [inline README.md:146]: I: Synk functionality was previously removed.  Removing mention from docs.

File: orphedomos-test/pom.xml [hunk 2]
```diff
@@ -26,9 +26,16 @@
                     <groupId>org.technologybrewery.orphedomos</groupId>
                     <artifactId>orphedomos-maven-plugin</artifactId>
                     <version>${project.version}</version>
-                    <configuration>
-                        <repoId>docker-hub</repoId>
-                    </configuration>
+                    <executions>
+                        <execution>
```
Comment (peter-mcclonski) [inline orphedomos-test/pom.xml:30]: I: Making login attempts only occur in testing during the deploy phase, for ease of testing.

Comment (peter-mcclonski) [reply orphedomos-test/pom.xml:30]: Q: May be worthwhile to provide a testing profile to always log in?  Minimal lift, but also minor value add.

Comment (d-ryan-ashcraft) [reply orphedomos-test/pom.xml:30]: We should probably do that with the Mojo test framework - which would be good for all functionality.  Would be a good follow on task to improve and keep quality consistent.

**Review by d-ryan-ashcraft — APPROVED** [review]

**Pull request merged**